src/core/Controls.ts

```diff
@@ -1,24 +1,75 @@
 import * as THREE from "three";
-
+import { Game } from "./Game";
 /**
  * An instance of this class perceives a user's actions within the game,
  * and controls the game input.
  */
 export class Controls {
-    private camera: THREE.PerspectiveCamera;
+    private static camera: THREE.PerspectiveCamera;
+    public static controls: any;
+    public static euler = new THREE.Euler( 0, 0, 0, 'YXZ' );
```

Comment:
why do we need `static` fields here?

Reply:
Because for errors without this, but i will check how to avoid it